Description: Add guarding commandline option around libutempter check
Author: Axel Beckert <abe@debian.org>
Forwarded: not-yet
Bug-Debian: https://bugs.debian.org/819781

--- a/configure.ac
+++ b/configure.ac
@@ -844,6 +844,8 @@
 ],
 [int x = DEAD_PROCESS; pututline((struct utmp *)0); getutent();], AC_DEFINE(GETUTENT), LIBS="$olibs")
 )
+AC_ARG_ENABLE(utempter, [  --enable-utempter       enable utempter support])
+if test "$enable_pam" = "yes"; then
 AC_CHECKING(ut_host)
 AC_TRY_COMPILE([
 #include <time.h>
@@ -860,6 +862,7 @@
   AC_DEFINE(HAVE_UTEMPTER)
   LIBS="$LIBS -lutempter"
 fi
+fi
 
 dnl
 dnl    ****  loadav  ****
